index.html

```diff
@@ -7,14 +7,19 @@
 
     <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css" />
     <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap-theme.min.css" />
+    <link rel="stylesheet" href="node_modules/font-awesome/css/font-awesome.min.css" />
 
 </head>
 <body>
 <header id="top" class="navbar navbar-default navbar-static-top navbar-inverse">
     <div class="container">
         <ul class="nav navbar-nav">
             <a class="navbar-brand" href="https://yagajs.org" data-placement="bottom" title="YAGA">YAGA</a>
+<<<<<<< HEAD
+            <li class="nav-item nav-link"><a href="https://leaflet-ng2.yagajs.org" title="Home"><span class="fa fa-cube" aria-hidden="true"></span>  leaflet-ng2</a></li>
+=======
             <li class="nav-item nav-link"><a href="http://leaflet-ng2.yagajs.org" title="leaflet-ng2"><span class="fa fa-cube" aria-hidden="true"></span>  leaflet-ng2</a></li>
+>>>>>>> develop
```

Comment:
We should remove the lines from the merge-conflict:
`+<<<<<<< HEAD`

`+=======`

`+>>>>>>> develop`